text/0000-clearer-changelogs.md

```diff
@@ -0,0 +1,142 @@
+
+## [2.0.110](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.0.109...gatsby@2.0.110) (2019-02-01)
+
+**Note:** Version bump only for package gatsby
+
+## [2.0.107](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.0.106...gatsby@2.0.107) (2019-01-31)
+
+### Bug Fixes
+
+- **gatsby:** always respond with index html without checking path ([#11400](https://github.com/gatsbyjs/gatsby/issues/11400)) ([2f79efe](https://github.com/gatsbyjs/gatsby/commit/2f79efe))
+- **gatsby:** check type when querying by ID ([#11448](https://github.com/gatsbyjs/gatsby/issues/11448)) ([1d95e67](https://github.com/gatsbyjs/gatsby/commit/1d95e67))
+
+### Features
+
+- **gatsby:** Allow specifying sort order per sort field ([#11381](https://github.com/gatsbyjs/gatsby/issues/11381)) ([11e8930](https://github.com/gatsbyjs/gatsby/commit/11e8930))
+
+---
+
+## Proposed changelog example:
+
+---
+
+## [2.0.112](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.0.111...gatsby@2.0.112) (2019-02-04)
+
+- 🐛 **Bug Fix** - **gatsby:** update terser-webpack-plugin to avoid webpack error [#11542](https://github.com/gatsbyjs/gatsby/issues/11542) ([@this-excellent-contributor](https://github.com/this-excellent-contributor))
+
+## [2.0.110](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.0.109...gatsby@2.0.110) (2019-02-01)
+
+- 🧹 **Chore**: update packages peer-dependencies to discard gatsby-alpha [#10252](https://github.com/gatsbyjs/gatsby/pull/10252)
+
+## [2.0.107](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.0.106...gatsby@2.0.107) (2019-01-31)
+
+- 🐛 **Bug Fix** - **gatsby:** always respond with index html without checking path [#11400](https://github.com/gatsbyjs/gatsby/issues/11400) ([@this-excellent-contributor](https://github.com/this-excellent-contributor))
+- 🐛 **Bug Fix** - **gatsby:** check type when querying by ID [#11448](https://github.com/gatsbyjs/gatsby/issues/11448) ([@this-excellent-contributor](https://github.com/this-excellent-contributor))
+
+  Fixes an `Expected value of type` error that could occur when running a GraphQL query that searches by Id.
+- 🚀 **Feature** - **gatsby:** Allow specifying sort order per sort field
+  [#11381](https://github.com/gatsbyjs/gatsby/issues/11381) ([@this-excellent-contributor](https://github.com/this-excellent-contributor))
+
+  When writing a GraphQL query that sorts on multiple fields, allow the sort order to be individually specified for each field.
+
+  See the [_Sort_ section of the GraphQL reference docs](https://www.gatsbyjs.org/docs/graphql-reference/#sort) for more information.
+
+-------
+
+# Motivation
+
+Refer to the [Motivation section of sibling RFC 27 for more info](https://github.com/gatsbyjs/rfcs/pull/27)
+
+The purpose of this RFC is to discuss making the information in Gatsby's changelogs more useful.
+
+# Detailed design
+
+Here's a list of changes implemented in the proposed example above.
```

Comment:
Question about this: are we able to define this template for existing tools? Or are we putting ourselves in a position to write custom changelog generation?

Reply:
We would need to create (probably fork from default we use now) changelog preset. Presets do some processing on commits and have handlebars templates.

I've been experimenting with adjusting default preset and this isn't very time consuming task (I would say few hours to implement these changes). Here's output of my forked preset https://github.com/pieh/lerna-playground/blob/master/packages/package-b/CHANGELOG.md (extra paragraph added to entry, credits to PR author, having actual entries for commits other than `feat:`, `fix:`).

Changelog preset I used is in https://github.com/pieh/conventional-changelog-pieh (no code review plz - it's messy PoC)

Reply:
Okay, cool. If it's not a huge amount of code to manage, I'm happy. I'd love to see better changelogs that are suited to our specific needs for sure.